designer/server/src/common/helpers/auth/csrf.js

```diff
@@ -13,6 +13,8 @@ const csrf = {
         plugin: crumb,
         options: {
           key: 'csrfToken',
+          logUnauthorized: true,
+          enforce: config.enforceCsrf || config.isProduction,
```

Comment:
If `config.enforceCsrf`, it'll fallback to `config.isProduction`. I don't think you want that?
Also, you already have this fallback logic in `config.ts`